windows/scripts/MatlabPostBuild.cmd

```diff
@@ -0,0 +1,12 @@
+set SOLUTION_DIR=%~1%
+set OUTPUT_DIR=%~2%
+
+echo MatlabPostBuild.cmd : copy matlab generated scripts to output.
+
+copy /y "%SOLUTION_DIR%..\matlab\+caffe\*.m" "%OUTPUT_DIR%matcaffe\+caffe"
```

Comment:
@mnidza OK, I removed the copy of those files, but I'm not sure either about what to do with the test files, because although it's quite simple not to copy the `+caffe\+test` folder, it's not so simple to exclude the `run_tests.m` file from the rest of `m` files of the same folder. According to this (http://stackoverflow.com/questions/4098444/move-all-files-except-some-file-pattern-from-a-dos-command) `xcopy` could be used instead of `copy`, but then a text file should be added which would include only that file to be excluded, which looks a bit strange to me. Wouldn't be good to copy those test files as well? I think they are quite useful to test in Matlab in an easy way if the compilation was good.